pkg/apis/app/v1alpha1/types.go

```diff
@@ -135,9 +136,10 @@ type FlinkClusterStatus struct {
 }
 
 type FlinkJobStatus struct {
-	JobID  string       `json:"jobID,omitEmpty"`
-	Health HealthStatus `json:"health,omitEmpty"`
-	State  JobState     `json:"state,omitEmpty"`
+	JobOverviewURL string       `json:"jobOverviewURL,omitEmpty"`
+	JobID          string       `json:"jobID,omitEmpty"`
+	Health         HealthStatus `json:"health,omitEmpty"`
+	State          JobState     `json:"state,omitEmpty"`
```

Comment:
Not entirely sure if I should be updating v1alpha1 anymore?

Reply:
I think we should leave v1alpha1 as is going forward.